client/state/themes/actions.js

```diff
@@ -420,6 +420,10 @@ export function installTheme( themeId, siteId ) {
 			themeId
 		} );
 
+		if ( isThemeFromWpcom( themeId ) ) {
+			dispatch( installWpcomParentTheme( themeId, siteId ) );
```

Comment:
> we should have function that returns parent id if there is one or null and later use that with standard `themeInstall`

I have a stashed patch that works more like this. I'll push it to a different PR and we can see which one we like best.